tests/e2e/wp/create.spec.ts

```diff
@@ -1,52 +1,150 @@
+
+      // Confirm created WP is in table
+      await wpDashboard.getReports();
+      await expect(wpDashboard.firstReport).toBeVisible();
+      const editNewWPButton = wpDashboard.firstReport.getByRole("button", {
+        name: "Edit",
+        exact: true,
+      });
+      await expect(editNewWPButton).toBeVisible();
+    } else {
+      await wpDashboard.copyoverButton.click();
+      await expect(wpDashboard.modal).toBeVisible();
+      await expect(wpDashboard.modal).toContainText("Continue");
+      await page
+        .getByRole("button", { name: "Continue from previous period" })
+        .click();
+
+      // Confirm created WP is in table
+      await wpDashboard.getReports();
+      await expect(wpDashboard.copiedReport).toBeVisible();
+      const editCopiedReportButton = wpDashboard.copiedReport.getByRole(
+        "button",
+        {
+          name: "Edit",
+          exact: true,
+        }
+      );
+      await expect(editCopiedReportButton).toBeVisible();
+    }
+  });
+
+  test("State user can fill and submit a Work Plan", async ({
+    page,
+    stateHomePage,
+    wpDashboard,
+    wpGeneralInformation,
+    wpTransitionBenchmarksProjections,
+    wpTransitionBenchmarkStrategy,
+    wpInitiativesInstructions,
+    wpInitiativesDashboard,
+    wpReviewAndSubmit,
+  }) => {
+    await logInStateUser(page);
+    await stateHomePage.wpButton.click();
+
+    // Dashboard
+    await wpDashboard.isReady();
     const editNewWPButton = wpDashboard.firstReport.getByRole("button", {
       name: "Edit",
       exact: true,
     });
-    await expect(editNewWPButton).toBeVisible();
-  } else {
-    await wpDashboard.copyoverButton.click();
-    await expect(wpDashboard.modal).toBeVisible();
-    await expect(wpDashboard.modal).toContainText("Continue");
-    await page
-      .getByRole("button", { name: "Continue from previous period" })
-      .click();
-
-    // Confirm created WP is in table
-    await wpDashboard.getReports();
-    await expect(wpDashboard.copiedReport).toBeVisible();
-    const editCopiedReportButton = wpDashboard.copiedReport.getByRole(
-      "button",
-      {
-        name: "Edit",
-        exact: true,
+    await editNewWPButton.click();
+
+    // General Information
+    await wpGeneralInformation.isReady();
+    await expect(wpGeneralInformation.disclosure).toBeVisible();
+
+    // Transition Benchmarks
+    await wpGeneralInformation.continueButton.click();
+    await wpTransitionBenchmarksProjections.isReady();
+    await wpTransitionBenchmarksProjections.editPopulations();
+
+    // Transition Benchmark Strategy
+    await wpTransitionBenchmarksProjections.continueButton.click();
+    await wpTransitionBenchmarkStrategy.isReady();
+    await wpTransitionBenchmarkStrategy.fillTextFields();
+
+    // Initiatives Instructions
+    await wpTransitionBenchmarkStrategy.continueButton.click();
+    await wpInitiativesInstructions.isReady();
+    await wpInitiativesInstructions.fillFormFields();
```

Comment:
looking at this flow I'm wondering if we'd consider renaming the "fill the page" option to something consistent across all pages?
`completeForm`, `fillForm`, `fillFormFields` etc.

Reply:
This is a very good point. I think I lost track of finesse as the days passed; I 100% support making the functions more consistent.